recipes/r-hdrcde/meta.yaml

```diff
@@ -1,94 +1,49 @@
+{% set version = '3.1' %}
+
 package:
   name: r-hdrcde
-  # Note that conda versions cannot contain -, so any -'s in the version have
-  # been replaced with _'s.
-  version: "3.1"
+  version: {{ version|replace("-", "_") }}
 
 source:
-  fn: hdrcde_3.1.tar.gz
+  fn: hdrcde_{{ version }}.tar.gz
   url:
-    - http://cran.r-project.org/src/contrib/hdrcde_3.1.tar.gz
-    - http://cran.r-project.org/src/contrib/Archive/hdrcde/hdrcde_3.1.tar.gz
-
-
-  # You can add a hash for the file here, like md5 or sha1
-  # md5: 49448ba4863157652311cc5ea4fea3ea
-  # sha1: 3bcfbee008276084cbb37a2b453963c61176a322
-  # patches:
-   # List any patch files here
-   # - fix.patch
+    - https://cran.r-project.org/src/contrib/hdrcde_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/hdrcde/hdrcde_{{ version }}.tar.gz
+  sha256: 9c86537f065167e90fab2a4adc2e87453b05c66fd71e6d33ea1ebc30ffedf608
+  md5: a364e2f5d0106bc4f4484cc18fc43ce5
```

Comment:
md5 is not needed if you have sha256. When you use conda skeleton from the newest conda-build, sha256 will be calculated automatically.